packages/fields/src/fields/featured-image/featured-image-edit.tsx

```diff
@@ -0,0 +1,122 @@
+/**
+ * WordPress dependencies
+ */
+import { Button, __experimentalGrid as Grid } from '@wordpress/components';
+import { useSelect } from '@wordpress/data';
+import { useCallback, useRef } from '@wordpress/element';
+// @ts-ignore
+import { MediaUpload } from '@wordpress/block-editor';
```

Comment:
I'm not sure I like the block-editor" dependency in the fields package. I guess it's because it's using a MediaUpload component that is specific to WordPress. Can we use `import { MediaUpload } from '@wordpress/media-utils';` instead ?

Reply:
TIL! Nice! Fixed with 7b1e40a6f0a56a990c97e45f1c57ffa12ba899d5.